### PR TITLE
Fix support for dashes in sample name

### PR DESCRIPTION
**Pull request by samnooij**
The mapped read count concatenation script was not prepared to handle sample names that contain dashes ('-'). It would just split file names by the last dash and use that as sample name. E.g. if you have sample names like "A-very-long-sample-name-1", it would return only the last part: "1". By itself that may not be such a big problem, but when the Snakemake rule `quantify_output` tries to merge the counts with the classifications based on sample name and scaffold name, this will fail and you will end up with a Sample_composition_graph.html file with 0 for all taxa.

TLDR:
In short, this patch solves the problem of faulty composition graphs for datasets in which sample names have dashes in them.

### CODE REVIEW

**Comment by florianzwagemaker** [comment]
I'm closing this particular pull request now without merging.
This change has been added through a standard commit: https://github.com/DennisSchmitz/Jovian/commit/0d8f21d03a3e947b34cbd8a850c46e299cb3eba6

In the past few weeks the project has been re-ordered in the `dev` branch. Meaning that the original file for this PR no longer exists in the same location/path which then again leads to a merge conflict.

Scripts directly functioning within the pipeline are now located in ` bin/scripts`
